home/storage/LocalStorage.js

```diff
@@ -60,6 +60,7 @@ async function getSessionAsync() {
 }
 
 async function saveSessionAsync(session) {
+  ExponentKernel.setSessionSecret(session.sessionSecret);
```

Comment:
Make this and the remove method awaitable so that if they fail for whatever reason (bridge fails, implementation becomes more complex and changes), we don't silently fail.